src/test/scala/cromwell/InvalidRuntimeAttributesSpec.scala

```diff
@@ -50,7 +48,7 @@ class InvalidRuntimeAttributesSpec extends CromwellTestkitSpec with BeforeAndAft
       )
     }
 
-    "fail on Local Backend" in {
+    /*"fail on Local Backend" in {
```

Comment:
Bah, thanks... I wanted to just run the one test to see why it was failing.  It shouldn't be commented out